app/models/user.rb

```diff
@@ -0,0 +1,30 @@
+class User < ApplicationRecord
+  has_one :user_profile
+  has_many :active_relationships, class_name: Relationship.name,
+    foreign_key: "follower_id", dependent: :destroy
+  has_many :passive_relationships, class_name: Relationship.name,
+    foreign_key: "followed_id", dependent: :destroy
+  has_many :following, through: :active_relationships, source: :followed
+  has_many :followers, through: :passive_relationships, source: :follower
+  has_many :markers
+  has_many :reviews
+  has_many :comments
+  has_many :likes
+  has_many :activities
+
+  devise :database_authenticatable, :registerable, :confirmable, :lockable,
```

Comment:
tách devise ra 1 pull riêng đi, 33 file nhiều quá

Reply:
Em tách ra làm 1 pull "Migrate and seed" và 1 pull là init_model cho liên quan rùi nhé :heart: